backend/handlers/bets/placebethandler.go

```diff
@@ -91,3 +65,24 @@ func PlaceBetHandler(loadEconConfig setup.EconConfigLoader) func(http.ResponseWr
 		json.NewEncoder(w).Encode(bet)
 	}
 }
+
+func checkUserBalance(user *models.User, betRequest models.Bet, sumOfBetFees int64, loadEconConfig setup.EconConfigLoader) error {
+	appConfig := loadEconConfig()
+	maximumDebtAllowed := appConfig.Economics.User.MaximumDebtAllowed
+
+	// Check if the user's balance after the bet would be lower than the allowed maximum debt
+	if user.AccountBalance-betRequest.Amount-sumOfBetFees < -maximumDebtAllowed {
+		return fmt.Errorf("Insufficient balance")
+	}
+	return nil
+}
+
+func createBet(username string, marketID uint, amount int64, outcome string) models.Bet {
+	return models.Bet{
+		Username: username,
+		MarketID: marketID,
+		Amount:   amount,
+		PlacedAt: time.Now(),
+		Outcome:  outcome,
+	}
+}
```

Comment:
I think we should make a models.NewBet() function or Bet.New() function that is public and replace this with the New function.

Reply:
Aren't models something that deal with how the database is actually defined?

Can a model merely mirror how a database is defined without defining the database itself?